mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/ui/game/ChessBoardRobot.kt

```diff
@@ -77,6 +77,21 @@ fun ChessBoardRobotInputScope.click(x: Int, y: Int, pointerId: Int = 0) {
   up(pointerId = pointerId)
 }
 
+/**
+ * Performs a click gesture with the pointer with the given id at a specific position in algebraic
+ * notation.
+ *
+ * @param x the row coordinate, as a character in algebraic notation.
+ * @param y the col coordinate, as an integer in algebraic notation.
+ * @param pointerId the identifier of the pointer.
+ */
+fun ChessBoardRobotInputScope.click(x: Char, y: Int, pointerId: Int = 0) {
+  down(x = x - 'a', y = 8 - y, pointerId = pointerId)
```

Comment:
What would you think about checking the inputs here ?

```suggestion
fun ChessBoardRobotInputScope.click(x: Char, y: Int, pointerId: Int = 0) {
  require(x in 'a'..'h') { "x out of valid range" }
  require(y in 1..8) { "y out of valid range." }
  down(x = x - 'a', y = 8 - y, pointerId = pointerId)
```

When moving at a certain position, the semantics are more or less clearly defined : position (-1, -1) is out of bounds, but it has a well-known location. I'm not sure where ('&', 5) would point to however ^^

Reply:
Also, you should maybe call the existing `click(x: Int, y: Int)` in the implementation of the function.

Reply:
Fixed in https://github.com/epfl-SDP/android/pull/309/commits/0e8a05757ab4ad799dd72f56141e8124bca0ffb2